src/algokit_utils/_debugging.py

```diff
@@ -189,17 +206,18 @@ def persist_sourcemaps(
         )
 
 
-def simulate_response(atc: AtomicTransactionComposer, algod_client: "AlgodClient") -> SimulateAtomicTransactionResponse:
-    """
-    Simulate and fetch response for the given AtomicTransactionComposer and AlgodClient.
-
-    Args:
-        atc (AtomicTransactionComposer): An AtomicTransactionComposer object.
-        algod_client (AlgodClient): An AlgodClient object for interacting with the Algorand blockchain.
-
-    Returns:
-        SimulateAtomicTransactionResponse: The simulated response.
-    """
+def simulate_response(
+    atc: AtomicTransactionComposer,
+    algod_client: "AlgodClient",
+    allow_more_logs: bool | None = None,
+    allow_empty_signatures: bool | None = None,
+    allow_unnamed_resources: bool | None = None,
+    extra_opcode_budget: int | None = None,
+    exec_trace_config: SimulateTraceConfig | None = None,
+    simulation_round: int | None = None,
+    skip_signatures: bool | None = None,  # noqa: ARG001 TODO: revisit
```

Comment:
TODO?

Reply:
removed, in case of simulate for creating the traces that argument will always be true